test/test_tripolar_grid.jl

```diff
@@ -0,0 +1,193 @@
+include("dependencies_for_runtests.jl")
+
+using Statistics
+using Oceananigans.Utils: get_cartesian_nodes_and_vertices
+using Oceananigans.ImmersedBoundaries: immersed_cell
+using Oceananigans.BoundaryConditions: Zipper
+
+using Oceananigans.Utils: KernelParameters, contiguousrange
+
+
+@kernel function compute_nonorthogonality_angle!(angle, grid, xF, yF, zF)
```

Comment:
What's a "nonorthogonality_angle"? Is this the deviation from orthogonality? (So it should be small)